corehq/apps/es/users.py

```diff
@@ -62,10 +62,16 @@ def show_only_inactive(self):
         return query.is_active(False)
 
 
-def domain(domain):
+def domain(domain, allow_mirroring=False):
```

Comment:
if there are already tests for this function it might be nice to add one for the new branch

Reply:
Thanks for the nudge, I'll add a test in a followup.

Reply:
https://github.com/dimagi/commcare-hq/pull/28218